packages/mux-player-react/src/lazy.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from 'react';
+
+import ConditionalSuspense from './ConditionalSuspense';
+import useIsBrowser from './useIsBrowser';
+import useIsIntersecting from './useIsIntersecting';
+
+import type { MuxPlayerProps, MuxPlayerRefAttributes } from './index';
+
+const MuxPlayerIndex = React.lazy(() => import('./index'));
+
+interface FallbackProps extends Omit<MuxPlayerProps, 'playerSoftwareVersion' | 'playerSoftwareName'> {
+  onIntersection?: () => void;
+}
+const Fallback = (props: FallbackProps) => {
+  const { style, onIntersection, ...rest } = props;
+
+  const intersectionRef = React.useRef<HTMLElement>(null);
+  const isIntersecting = useIsIntersecting(intersectionRef);
+
+  useEffect(() => {
+    if (isIntersecting && onIntersection) {
+      onIntersection();
+    }
+  }, [isIntersecting, onIntersection]);
+
+  return (
+    <>
+      {/* TODO: can we add mux-player to JSX.IntrinsicElements */}
+      {/* @ts-ignore */}
+      <mux-player
```

Comment:
shouldn't this import the React MuxPlayer? with the correct prop / event mapping etc

Reply:
So this is just the placeholder. And the placeholder should (in theory, [not quite in practice](https://mux.slack.com/archives/C032TQ30ARL/p1664827962292839?thread_ts=1664812026.713469&channel=C032TQ30ARL&message_ts=1664827962.292839)) exist only when code for mux-player and mux-player-react is still loading. And, of course, since that js is still loading, a `mux-player` element is basically just a div. 

The reason we use a `mux-player` html element here is not to have any of player's functionality, but rather, to inherit any styles targeted at `mux-player`.

Reply:
cool makes sense

Reply:
Is it worth noting this down in a comment in source?

Reply:
For sure. Unless we ditch this approach and use a wrapper 🤔
https://github.com/muxinc/elements/pull/414#discussion_r987016810

Reply:
Not a problem now that we have a wrapper.